app/controllers/concerns/last_edited_by.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module LastEditedBy
+  extend ActiveSupport::Concern
+
+  protected
+
+  # Updates the attribute "last_edited_by" with the current user
+  # The resource can be given as a parameter this method uses the
+  # `last_edited_by_resources` method that can be defined in the
+  # controller
+  #
+  # @params resource: a model having the last_edited_by attribute
+  def update_last_edited_by(resource = nil)
```

Comment:
This method does seem pretty complicated. Couldn't you just pass in the resource?

Reply:
It was indeed a bit too complicated for what it was doing, I simplified it :)

Reply:
Nice!